src/clr-angular/forms/radio/radio-container.ts

```diff
@@ -19,13 +18,13 @@ import { NgControlService } from '../common/providers/ng-control.service';
   template: `
     <ng-content select="label"></ng-content>
     <label *ngIf="!label && addGrid()"></label>
-    <div class="clr-control-container" [ngClass]="controlClass()">
-      <div [class.clr-radio-inline]="clrInline">
-        <ng-content select="clr-radio-wrapper"></ng-content>
+    <div class="clr-control-container" [class.clr-control-inline]="clrInline" [ngClass]="controlClass()">
```

Comment:
Since we moved `[class.clr-control-inline]="clrInline"`, don't we need to add tests for this?

Reply:
Added a test